components/service/glean/src/main/java/mozilla/components/service/glean/TimingDistributionMetricType.kt

```diff
@@ -34,11 +34,11 @@ data class TimingDistributionMetricType(
     private val logger = Logger("glean/TimingDistributionMetricType")
 
     // Holds the Job returned from launch{} for awaiting purposes
+    @Volatile
```

Comment:
Actually, sorry, let's remove this. This was an attempt fix for the intermittent, but it's not working as intended. Let's settle with what @mdboom finds.